components/description/description.module.scss

```diff
@@ -0,0 +1,78 @@
+@import "../../styles//media.scss";
+
+
+.container {
+  background: #fff;
+  height: 371px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 0px 5px 1px rgb(0,0,0,0.3);
+  position: relative;
+  @media only screen and (max-width: $mobile) {
+    height: 409px;
+    flex-direction: column;
+    overflow: inherit;
+    // justify-content: center;
+    // align-items: center;
+  }
+}
+
+.content {
+  margin-right: 143px;
+  @media only screen and (max-width: $mobile) {
+    padding: 24px 16px;
+    margin-right: 0;
+  }
+
+  @media only screen and (max-width: $tablet) {
+    padding: 24px 16px;
+    margin-right: 0;
+  }
+}
+
+.title {
+  font-size: 36px;
+  font-weight: 500
+  ;
```

Comment:
Please fix, use prettier.

Reply:
I think prettier runs automatically on every commit. I'll check why it didn't catch this (maybe we only run it on tsx files and not CSS)

Reply:
@urish Thanks :)

Reply:
Done! 
d819918fc581a8e1b03acfc5133d2faea56717d0